### DISCUSSION TITLE
Collection Blueprint .yaml file is not updated when a Taxonomy is configured to allow that Collection

### DISCUSSION THREAD

**Discussion by ryanvelbon** [statamic/cms#6448] [Troubleshooting]
_Posting as a discussion in case it is not an issue but just me._

When I did the following:

1. Create a `Articles` collection
2. Create a `Categories` taxonomy
3. Configure `Categories` Taxonomy > Content Model > Collections > `Articles`

The file `resources/blueprints/collections/articles/articles.yaml` is only updated to include the new taxonomy after visiting `http://example.com/cp/collections/articles/blueprints/articles/edit` and clicking Save.

Perhaps the `please` CLI has some command which rebuilds certain files when updates are made?

**Comment by jasonvarga** [comment]
The taxonomy fields are automatically added to the blueprint even if they aren't in the file. 

Try it. Instead of hitting save on the blueprint, just open up an entry. You'll see the new terms field.
